modelcenter/ERNIE-3.0/fastdeploy_en.md

```diff
@@ -0,0 +1,50 @@
+## 0. FastDeploy
+
+FastDeploy is an Easy-to-use and High Performance AI model deployment toolkit for Cloud, Mobile and Edge with out-of-the-box and unified experience, end-to-end optimization for over 150+ Text, Vision, Speech and Cross-modal AI models. FastDeploy Supports AI model deployment on
+**X86 CPU、NVIDIA GPU、ARM CPU、XPU、NPU、IPU** etc. You can switch different inference backends and hardware with a single line of code.
+
+Deploying AI model in 3 steps with FastDeploy: (1)Install FastDeploy SDK;  (2)Use FastDeploy's API to implement the deployment code;  (3) Deploy.
+
+**Notes** : This document downloads FastDeploy examples to complete the high performance deployment experience; only X86 CPUs, NVIDIA GPUs are shown for reasoning and GPU environments are ready by default (e.g. CUDA >= 11.2, etc.), if you need to deploy AI model on other hardware or learn about FastDeploy's full capabilities, please refer to [FastDeploy GitHub](https://github.com/PaddlePaddle/FastDeploy).
+
+## 1. Install FastDeploy SDK
+```
+pip install fastdeploy-gpu-python==0.0.0 -f https://www.paddlepaddle.org.cn/whl/fastdeploy_nightly_build.html
+```
+## 2. Run Deployment Example
+```
+# Download Deployment Example
+git clone https://github.com/PaddlePaddle/FastDeploy.git
+cd  FastDeploy/examples/text/ernie-3.0/python
+
+#  Download the fine-tuned ERNIE 3.0 model for the AFQMC dataset
```

Comment:
Download the fine-tuned ERNIE 3.0 model trained from the AFQMC dataset

Reply:
ok